.github/workflows/ci.yml

```diff
@@ -76,8 +76,15 @@ jobs:
           php maintenance/update.php --quick
 
       - name: Run PHPUnit
+        run: php tests/phpunit/phpunit.php -c extensions/DataAccounting
+        if: matrix.mw != 'REL1_37'
+
+      - name: Run PHPUnit with code coverage
         run: |
-          php tests/phpunit/phpunit.php -c extensions/DataAccounting
+          php tests/phpunit/phpunit.php -c extensions/DataAccounting/ --coverage-clover coverage.xml
```

Comment:
I think there is a GH Actions module for coverage badge. Should we use Actions instead?

Reply:
I have not seen that approach yet. Do you think it is better in some manner than these two lines?

Reply:
I actually have just read https://github.com/codecov/codecov-action. It is only convenient to use Actions if more info needs to be provided to codecov.io, e.g. commit hash. Probably overkill with Actions.